packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -69,6 +77,12 @@ class UnThemedLogRow extends PureComponent<Props, State> {
     });
   };
 
+  onChangeHoverBackground = (bool: boolean) => {
```

Comment:
Let's add some comment here why manually triggering the hover is necessary right now.

Reply:
Good point!